docs/sdkx-react-native/tracking.mdx

```diff
@@ -0,0 +1,170 @@
+---
+sidebar_position: 90
+title: Tracking
+description: "Track events and user actions when the user starts a new conversation. Attach custom metadata to every conversation started via the SDK."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Tracking {#tracking}
+
+<Intro>
+
+Track events and user actions when the user starts a new conversation.
+Attach custom metadata to every conversation started via the SDK.
+
+</Intro>
+
+## Adding Tags to Conversations {#adding-tags}
+
+You can attach tags while reporting an issue by passing them to the `tags` parameter at the time of calling `showConversation` function. You can pass an **array of strings** which will get added as `tags` when the issue is created.
+
+Pass this Map to `showConversation` API. For eg:
+
+```javascript
+const apiConfig = {
+    tags: {"foo", "bar"},
+};
+showConversation(apiConfig);
+```
+
+Similarly, you can pass this configuration to other APIs (ie. `showConversation`, `showFAQsWithConfig`, `showFAQSectionWithConfig`).
+
+<Admonition type="info" title="Note">
+
+**On tag names & compatibility**
+
+- Tags must be pre-created in the Helpshift Dashboard (Settings → Tags), otherwise, they will be ignored.
+- The attached tags must exactly match the tags present on the dashboard.
+
+</Admonition>
+
+## Set Custom Metadata {#set-custom-metadata}
+
+If you want to set Custom Metadata at the time of Issue creation, follow the steps.
+
+1. Define your custom metadata `Map` as key-value pairs.
+2. Add the custom metadata `Map` in the map expected in the `Parameters` argument with `customMetadata` as the key.
+
+Pass this Map to the following APIs:
+
+`showConversation` API,
+
+```javascript
+const CIFs = { stock_level: { type: 'number', value: '1505' } };
```

Comment:
the code samples here are incorrect i think.
The sample is for CIF but the section is of custom meta data